assets/extensions/main.js

```diff
@@ -38,28 +35,66 @@ const Main = () => {
 		);
 	}
 
+	const freeExtensions = extensions.filter(
+		( extension ) => extension.price === 0
+	);
+	// TODO: Specify which plugins are third party
+	const thirdPartyExtensions = extensions.filter(
+		( extension ) => extension.price === 0
+	);
+	const installedExtensions = extensions.filter(
+		( extension ) => extension.is_installed
+	);
+
+	const tabs = [
```

Comment:
We could add a `content` in these objects, like: `content: <AllExtensions extensions={ extensions } />` / `content: <FilteredExtensions extensions={ freeExtensions } />`, so we could just make an array map to list the routes. WDYT?

Reply:
Good idea! ea919aef98aee724050f16423d0736d6eff6bb65